### ISSUE TITLE
Добавить включение\отключение нагрузки по временным рамкам.

### ISSUE THREAD

**Issue by OdessaEyes** [kmaximv/Wi-Fi-Sensor#26]
Надеюсь будет возможность добавить в прошивку данную функцию. Как вариант указывать временные рамки включения\отключения или время включения\выключения + задержка.
Хочу использовать Ваш проект для фитолампы. Спасибо.

**Comment by kmaximv** [comment]
Да, эта функция в планах, но по срокам пока ничего не могу сказать. Времени на всё не хватает.

**Comment by OdessaEyes** [comment]
Внесение в список to do уже радует. Буду с нетерпением ждать. Спасибо.
